README.md

```diff
@@ -453,7 +455,7 @@ see how we would make use of `help`
 **example**
 
 ```js
-// NOTE: we've import desc and help in addition to previous imports
+// NOTE: we'll import desc and help in addition to previous imports
```

Comment:
I could've also changed this to be `we've imported desc ...` but I'm not sure which one you prefer, so I just but a future wording here. You can be the judge of which wording is better, and I'm open to changing it :smile: